libflux/src/flux/semantic/builtins.rs

```diff
@@ -275,6 +275,267 @@ pub fn builtins() -> Builtins<'static> {
                 "splitRegexp" => Node::Builtin("forall [] (r: regexp, v: string, i: int) -> [string]"),
                 "getString" => Node::Builtin("forall [] (r: regexp) -> string"),
             }),
+            "universe" => Node::Package(maplit::hashmap! {
+                "inf" => Node::Builtin("forall [] duration"),
+                "int" => Node::Builtin("forall [t0] (v: t0) -> int"),
```

Comment:
We expect to get a runtime error if we can't parse the type that's passed in as an integer, correct?

Reply:
Yes, that's right.  In that case the current signature is ok, yes?

Reply:
Yeah just making sure.